srtcore/packet.h

```diff
@@ -115,32 +115,28 @@ class IOVector
     inline void setLength(size_t length)
     {
 #ifdef _WIN32
-        len = (ULONG) length;
+        len = (ULONG)length;
 #else
         iov_len = length;
 #endif
     }
 };
 
-
 /// To define packets in order in the buffer. This is public due to being used in buffer.
 enum PacketBoundary
 {
     PB_SUBSEQUENT = 0, // 00
```

Comment:
By this cleanup you could also add some comment here. For example: packet between the first and last packet of a message.